x-pack/solutions/security/plugins/security_solution/common/siem_migrations/model/api/rules/rule_migration.schema.yaml

```diff
@@ -396,6 +396,26 @@ paths:
                 additionalProperties:
                   $ref: '../../rule_migration.schema.yaml#/components/schemas/PrebuiltRuleVersion'
 
+  /internal/siem_migrations/rules/{migration_id}/integrations:
```

Comment:
Do we have the {migration_id} in the path? I am confused

Reply:
This needs to be `/internal/siem_migrations/rules/integrations`